azurerm/internal/services/servicebus/resource_arm_servicebus_subscription.go

```diff
@@ -222,6 +236,7 @@ func resourceArmServiceBusSubscriptionRead(d *schema.ResourceData, meta interfac
 		d.Set("requires_session", props.RequiresSession)
 		d.Set("forward_to", props.ForwardTo)
 		d.Set("forward_dead_lettered_messages_to", props.ForwardDeadLetteredMessagesTo)
+		d.Set("status", helper.FlattenEntityStatus(props.Status))
```

Comment:
Similar to the above comment, we should aim for simplicity here.

```suggestion
		d.Set("status", string(props.Status))
```

Reply:
Changed this to a one-liner in 87b1115c70dd1df99865cd7288d0627b7c751842